<launch>

    <arg name="markerSize"      default="0.05"/>    <!-- in m -->
    <arg name="side"             default="left"/>
    <arg name="ref_frame"       default="base_link"/>  <!-- leave empty and the pose will be published wrt param parent_name -->
    <arg name="camera_info"      default="/camera_image/camera_info"/>
    <arg name="image"           default="/camera_image/image_raw"/>
    <arg name="camera_frame"     default="camera_link"/>

    <node pkg="aruco_ros" type="marker_publisher" name="aruco_marker_publisher">
        <remap from="/camera_info"       to="$(arg camera_info)" />
        <remap from="/image"             to="$(arg image)" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="$(arg camera_frame)"/>
    </node>

</launch>

